apollo-router/src/spec/query.rs

```diff
@@ -81,7 +81,12 @@ impl Query {
                                 },
                             );
 
-                            response.errors.extend(parameters.errors.into_iter());
+                            if !parameters.errors.is_empty() {
+                                response.extensions.insert(
+                                    "formatting",
```

Comment:
Is "formatting" the correct name? Would "composition" be a better choice or "assembly". Is there an established convention for this kind of naming?
UPDATE: I just saw the other comments about naming. I don't think "nullability" is a good name, but could be convinced.

Reply:
Possibly `value-completion`? As this is the part of the spec that handles nullability https://spec.graphql.org/June2018/#sec-Value-Completion

Reply:
cc @Geal before you squashmerge ^^

Reply:
that works for me